MainModule/Server/Core/Remote.lua

```diff
@@ -547,6 +548,21 @@ return function(Vargs, GetEnv)
 							warn = function(...) local nums = {...} for i,v in pairs(nums) do Remote.Terminal.LiveOutput(p,"WARN: "..tostring(v)) end end;
 						})
 
+						if not server.Remote.RemoteExecutionConfirmed[p.UserId] then
```

Comment:
this area feels unnecessary as I don't think anyone is really getting tricked into running the terminal and executing code that would be stopped by a prompt...

Reply:
It makes spoofing it harder. But of course it's still very possible to spoof.